### DISCUSSION TITLE
dbt Installation

### DISCUSSION THREAD

**Discussion by kOlsonTA** [OpenLineage/OpenLineage#406] [Q&A]
I've been trying to test OpenLineage on my dbt project, but there seems to be an issue with the pip install. 
I started with a working dbt project. 
I ran: pip3 install openlineage-dbt
Then I try to using "dbt-ol run" and get this error: 
Program 'dbt-ol' failed to run: No application is associated with the specified file for this operationAt line:1 char:1
+ dbt-ol run
+ ~~~~~~~~~~.
At line:1 char:1
+ dbt-ol run
+ ~~~~~~~~~~
    + CategoryInfo          : ResourceUnavailable: (:) [], ApplicationFailedException
    + FullyQualifiedErrorId : NativeCommandFailed

So I navigate to ... scoop\apps\python\current\Scripts and to check to see if dbt-ol is there. Here I find a "dbt-ol" with no file extension. This file when opened is a python script but not a compiled exe. 

I'm a little new at the whole python/pip thing so if this is an easy fix I apologies ahead of time, but I could really use some help, or at least confirmation that the pip install is busted.

**Comment by mobuchowski** [comment]
Hey, are you using Windows?
I must say I've never seen similar error.

**Reply by kOlsonTA** [reply]
Yep using Windows. Used pip to install dbt and adapters for it without a problem.

**Reply by kOlsonTA** [reply]
Is the exe something I can download directly?

**Reply by mobuchowski** [reply]
Thanks for the info. I think this is on us, we have slightly different packaging setup than dbt, and this can cause this problem. I've created the issue and will try to fix it. https://github.com/OpenLineage/OpenLineage/issues/408

Python programs generally do not use binary executable files, unless created with external program like [pyinstaller](https://www.pyinstaller.org/). We're generally trying to match dbt setup.

**Reply by kOlsonTA** [reply]
Thanks for looking into it, I appreciate the help.
